tasty/src/dotty/tools/tasty/TastyFormat.scala

```diff
@@ -486,6 +485,9 @@ object TastyFormat {
   final val EMPTYCLAUSE = 45
   final val SPLITCLAUSE = 46
 
+  // Best Effort TASTy only
+  final val ERRORtype = 50
```

Comment:
For the same reason, `ERRORtype` should be confined to `BestEffortTastyFormat`. It should not appear in the public `TastyFormat` library.